src/TarHeader.php

```diff
@@ -124,7 +124,7 @@ protected function packFields($fields) {
 	 * Generate unsigned checksum of header
 	 *
 	 * @param string $header
-	 * @return string unsigned checksum
+	 * @return float|int unsigned checksum
```

Comment:
This function does actually return a number (usually an int, but if the number is big, then it could end up being a float.